app/templates/base.html

```diff
@@ -45,6 +45,10 @@
                     <li class="nav-item">
                         <a class="nav-link" href="/search">Search</a> 
                     </li>
+                    <li class="nav-item">
+                        <button onclick="getDailyJoke()" class="btn btn-link" style='text-decoration:none;'>Get a Joke</button>
```

Comment:
Prefer using `addEventHandler`

Reply:
(This one is still relevant)

Reply:
I am trying to do so.
I didn't succeed yet.

Reply:
I have changed everything according to your requests :)
I  added an event listener